api/app/services/pdfs/expenses_overview_service.rb

```diff
@@ -0,0 +1,220 @@
+# frozen_string_literal: true
+
+require 'prawn'
+
+module Pdfs
+  class ExpensesOverviewService
+    include Prawn::View
+    include Pdfs::PrawnHelper
+
+    COLOR_GREY = 'DDDDDD'
+
+    TABLE_HEADER = [
+      {
+        content: I18n.t('activerecord.attributes.user.id'),
+        background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('activerecord.attributes.service_specification.name'),
+        background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('pdfs.expense_sheet.info_block.header.expense_sheet_time_duration.label'),
+        background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('activerecord.attributes.expense_sheet.work_days.other'),
+        colspan: 2, background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('activerecord.attributes.expense_sheet.workfree'),
+        colspan: 2, background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('activerecord.attributes.expense_sheet.sickness'),
+        colspan: 2, background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('activerecord.attributes.expense_sheet.paid_vacation_days.other'),
+        colspan: 2, background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('activerecord.attributes.expense_sheet.unpaid_vacation_days.other'),
+        colspan: 2, background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('activerecord.attributes.expense_sheet.way_expenses'),
+        background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('activerecord.attributes.expense_sheet.clothing'),
+        colspan: 2, background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('pdfs.expense_sheet.expense_table.row_headers.extra'),
+        background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('pdfs.expense_sheet.expense_table.headers.full_amount'),
+        colspan: 2, background_color: COLOR_GREY, align: :right
+      }
+    ].freeze
+
+    TABLE_SUB_HEADER = [
```

Comment:
Ich hetti die Konstante in es separats file gno, dass es file hesch wie https://github.com/stiftungswo/better-izivi/blob/develop/api/app/services/pdfs/expense_sheet/fields/expense_table.rb
Macht das File chli übersichtlicher